src/ebay-listbox-button/listbox-button.tsx

```diff
@@ -286,7 +286,7 @@ const ListboxButton: FC<EbayListboxButtonProps> = ({
                         {updatelistBoxButtonOptions}
                     </div>
                 </div>}
-            <select hidden className="listbox-button__native" name={name} value={selectedOption.props.value}>
+            <select hidden className="listbox-button__native" name={name} defaultValue={selectedOption.props.value}>
```

Comment:
The select value needs to be in sync with the selectedOption state variable. Unfortunately we can't use defaultValue since it doesn't update. We should use `readOnly` instead but select elements doesn't support it. I think we should use an `input type="text" hidden` with `readOnly` instead here.